### ISSUE TITLE
popup in bottom right of card 'Live: Camera Name' continously pops up on cards

### ISSUE THREAD

**Issue by inittabayanami** [dermotduffy/frigate-hass-card#1203]
**Checklist:**

- [X ] I updated to the latest version available
- [X ] I cleared the cache of my browser

**Release with the issue:**
v5.1.1
**Last working release (if known):**

**Browser and Operating System:**
Firefox Windows 11

**Description of problem:**
I have multiple frigate cards setup on a 'Cameras' Dashboard to have a grid view of cameras, when first loading this dashboard or coming back to it, (also sometimes when just looking at it) all of the frigate cards in a row will flash up the 'Live: Camera name' baloon popup in the bottom right, it will cycle through the cards and repeat but then after awhile stop. this can be seen at: https://www.youtube.com/watch?v=lgmZlO9FVrE

**Frigate card diagnostic information**

```{
  "ha_version": "2023.6.2",
  "card_version": "5.1.1",
  "browser": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/114.0",
  "date": "2023-06-18T17:33:31.307Z",
  "frigate_version": {
    "ffac51088eeb9acea126512d1fcf348e": "4.0.0/0.12.1-367d724"
  },
  "lang": "en",
  "timezone": "America/New_York",
  "git": {
    "build_version": "5.1.1-HEAD+g2d977be",
    "build_date": "Sat, 20 May 2023 20:35:29 GMT",
    "commit_date": "Sat, 20 May 2023 13:32:32 -0700"
  },
  "config": {
    "type": "custom:frigate-card",
    "cameras": [
      {
        "camera_entity": "camera.driveway"
      }
    ],
    "view": {
      "default": "live",
      "update_force": true
    },
    "menu": {
      "buttons": {
        "recordings": {
          "enabled": true,
          "alignment": "matching",
          "priority": 100,
          "icon": "mdi:air-filter"
        }
      }
    },
    "live": {
      "layout": {
        "fit": "fill"
      }
    }
  }
}

```

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**

**Comment by dermotduffy** [comment]
Thanks for the video. As a workaround you can disable those popup titles with:

```yaml
live:
  controls:
    title:
      mode: none
```

As for the cause this suggests the media is continually loading & reloading. Do the camera views load okay in HA itself without the card? (i.e. if you just look at the camera entity?).

If you have `go2rtc` configured, you could try the `go2rtc` live provider and see if that works?

```yaml

type: custom:frigate-card
cameras:
  - camera_entity: camera.driveway
    live_provider: go2rtc
```

**Comment by inittabayanami** [comment]
no issues with viewing the camera with the camera card.
Workaround works great, Thanks, I felt like I was watching a game of whack a mole every time I checked cameras.
I do not currently have go2rtc setup but I will do some research and give that a shot.

**Comment by dermotduffy** [comment]
> Workaround works great, Thanks, I felt like I was watching a game of whack a mole every time I checked cameras.

Look like a crazy and unwelcome disco effect!

No idea why this was happening to you in the first place, though. That popup fires every time the video element fires the `loadeddata`  event, which is supposed to be only the successful load of the first frame. It repeatedly happening for you suggests something is continually reseting the video.

In your video clip, I notice the time on the top-left camera appears to 'stick' at :17. Does make me wonder if there's something wrong with your video feed somehow causing it to continually reset and it's just not noticeable until something like the title popups trigger continually.

TL;DR: Not sure if this is an issue with the card, or with your setup, so unclear what action I should take on this end :-(

**Comment by dermotduffy** [comment]
I've added a troubleshooting entry for others who run into this issue. I'll close this for now, but if anyone else has this problem and has a clue as to why it happens, please update the issue!
